packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| coerce-invalid          | Boolean - Whether to coerce invalid max/min values on blur.                                                               |
+| maximum-fraction-digits | Maximum number of digits after the decimal separator.                                                                     |
+| minimum-fraction-digits | Minimum number of fraction digits. Values with fewer digits will be right-padded with zeros to meet the specified length. |
+
+### Behavior
+
+#### Mouse Interactions
+
+Hovering over the field input indicates interactivity.
+
+Clicking on the field input focuses the input and places a caret. If no value
+exists, the caret is placed at the far left. Otherwise, it is placed where
+clicked.
+
+![mouse interactions](./spec-images/localized-number-input-cursor-click.png)
+
+#### Keyboard Interactions
+
+| Key           | Description                                                                                                                                      |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ENTER`/`ESC` | No action.                                                                                                                                       |
+| `UP ARROW`    | (If `hide-step = false`) Increment the number by the step value. (If `hide-step = true`) Moves the caret to the beginning of the entered number. |
+| `HOME`        | Moves the caret to the beginning of the entered number.                                                                                          |
+| `DOWN ARROW`  | (If `hide-step = false`) Decrement the number by the step value. (If `hide-step = true`) Moves the caret to the end of the entered number.       |
+| `END`         | Moves the caret to the end of the entered number.                                                                                                |
+| _Numbers_     | (If focused) Adds number at cursor.                                                                                                              |
+| `SPACE`       | No action.                                                                                                                                       |
+| _Alphabetic_  | No action.                                                                                                                                       |
```

Comment:
Some numeric inputs allow entering `e` and `E` and then will expand inputs like `1e3` to `1000`. The nimble input today allows you to enter e/E but then rejects the input on blur if it's present. Should we fix that bug as part of this work? Or add the feature to expand inputs?

Reply:
So, the `nimble-number-field` currently handles entering `e`, but not `E`, and expands the number as expected. That is, if you type `1e3` that will automatically be changed to 1000 if you have `max` either not set, or set to at least 1000.

Reply:
Huh, must be browser specific. So far I've tried macOS Firefox and Windows Firefox and Edge, and typing "1e3" and then click/tab outside results in the value going back to what it was on focus.

Reply:
Proposing that we do not support scientific notation. Added row to that effect.